### PR TITLE
INDY-902: sdk test integration in node

### PR DESCRIPTION
**Pull request by ArtObr**

### CODE REVIEW

File: indy_client/test/cli/test_nym.py [hunk 1]
```diff
@@ -1,365 +1,114 @@
...
 
 
 @pytest.fixture("module")
-def trustAnchorWallet(trustAnchorSigner):
-    w = Wallet(trustAnchorSigner.identifier)
-    w.addIdentifier(signer=trustAnchorSigner)
-    return w
-
-
-def testPoolNodesStarted(poolNodesStarted):
```
Comment (dsurnin) [inline indy_client/test/cli/test_nym.py]: Do not delete this test

File: indy_client/test/cli/test_nym.py [hunk 2]
```diff
@@ -1,365 +1,114 @@
...
-
-    do('send NYM dest={}'.format(idr))
-
-    do('new key with seed {}'.format(poolTxnData['seeds']['Steward1']))
-
-    do('send NYM dest={}'.format(idr), within=3,
-       expect=["Nym {} added".format(idr)])
-
-
-def test_send_same_nyms_only_first_gets_written(
```
Comment (dsurnin) [inline indy_client/test/cli/test_nym.py]: Rewrite this test

File: indy_client/test/cli/test_nym.py (hunk 1 above)
Comment (ArtObr) [reply indy_client/test/cli/test_nym.py]: Test rewritten

File: indy_client/test/cli/test_nym.py (hunk 2 above)
Comment (ArtObr) [reply indy_client/test/cli/test_nym.py]: Test rewritten

**Pull request merged**